awscli/examples/oam/delete-sink.rst

```diff
@@ -0,0 +1,8 @@
+**To Delete a sink**
+
+The following ``delete-sink`` example deletes a sink. You must delete all links to a sink before you can delete that sink. ::
+
+    aws oam delete-sink \
+        --identifier arn:aws:oam:us-east-2:123456789012:sink/a1b2c3d4-5678-90ab-cdef-example12345
+
```

Comment:
Same here. There is a missing output section. If this command produces no output please add the appropriate line.
https://aws.github.io/aws-cli/docs_styleguide.html#output